tools/wake/main.cpp

```diff
@@ -553,7 +553,11 @@ int main(int argc, char **argv) {
   }
 
   if (failed) {
-    describe(db.failed(verbose || tag), script, debug, verbose, tag);
+    if (verbose) {
+      describe(db.failed(verbose || tag), script, debug, verbose, tag);
+    } else {
+      describe_failed(db.failed_output());
```

Comment:
So when I wrote this I wasn't thinking about how to do this properly and I've since thought about how to do this a bit more. There are a collection of "database commands" in wake of which --failed is only one. I think we should have 3 modes of operation instead

1) this one (shows command and stdout/stderr but nothing else)
2) the old metadata format
3) the verbose mode

but this should work with all queries (e.g. --input, --output, --last, etc....)

Reply:
Can you confirm the API? Are you thinking something like

`wake $DATABASE_COMMAND $DATABASE_MODE`

where command is `--input`, `--output`, `--failed`, `--last` and so on
and mode is `--verbose`, `--metadata`, `--std-streams`(default), (std-streams is a bad name, please pitch better ones)

so then you do one of  
```bash
wake --failed --metadata # old style
wake --last # or 
wake --failed --std-streams # for the default of this PR
wake --output --verbose # for the verbose mode
```

Reply:
Yeah and keep in mind that you can combine those (by intersection). So for instance `wake --failed --last` would show in this new style, the failed jobs from the last invocation of wake. Further more you could combine it to `wake --failed --last --input my_input_file.txt` and it would show you only the jobs that failed, from the last invocation of wake, that also read my_input_file.txt as an input.

Reply:
basically I want it to work like it does today, but with a new output style as the default that will be more helpful to the average developer (and me quite frankly)

Reply:
Indeed it is a union

```
  if (last) {
    describe(db.last(verbose || tag), script, debug, verbose, tag);
  }

  if (failed) {
    describe(db.failed(verbose || tag), script, debug, verbose, tag);
  }
```

We should probably add an intersection for last-failed I think but we can do that in a follow up PR

Reply:
intersections for the other options aren't as helpful